Tanium bug fixed (#10134)
demisto/content: Packs/Tanium/ReleaseNotes/1_0_4.md

#### Integrations
##### Tanium v2

> - Fixed an issue that not handled properly the '=', ';' characters when input into the value on "parameters"
> - Upgraded the Docker image to demisto/python3:3.8.6.13358.

Reviewer: ```suggestion
- Fixed an issue where parameters were not handled properly if contained '=' or ';' characters,
- Upgraded the Docker image to demisto/python3:3.8.6.13358.
```
